pulp_2to3_migration/app/migration.py

```diff
@@ -139,6 +150,87 @@
                 pb.save()
 
 
+async def migrate_distributors(plan):
+    """
+    A coroutine to migrate pre-migrated distributors.
+
+    Args:
+        plan (MigrationPlan): Migration Plan to use.
+    """
+    # gather all needed plugin distributor migrators
+    distributor_migrators = {}
+    plugins_to_migrate = plan.get_plugins()
+
+    for plugin, plugin_migrator in PLUGIN_MIGRATORS.items():
+        if plugin not in plugins_to_migrate:
+            continue
+        distributor_migrators.update(**plugin_migrator.distributor_migrators)
+
+    progress_data = dict(
+        message='Migrating distributors to Pulp 3', code='migrating.distributors', total=0
+    )
+    with ProgressReport(**progress_data) as pb:
+        pulp2distributors_qs = Pulp2Distributor.objects.filter(
+            pulp3_distribution=None,
+            pulp3_publication=None,
+            not_in_pulp2=False)
+        total_pulp2distributors = pulp2distributors_qs.count()
+        pb.total += total_pulp2distributors
```

Comment:
This works fine but it should probably just be normal assignment

Reply:
good catch, that's the leftovers from my older versions of the code, i will remove that variable